compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/Symbols.java

```diff
@@ -156,7 +157,12 @@ public static BTypeSymbol createTypeSymbol(int symTag,
                                                Location pos,
                                                SymbolOrigin origin) {
         if (type != null && type.tag == TypeTags.INVOKABLE) {
-            return createInvokableTypeSymbol(symTag, flags, pkgID, type, owner, pos, origin);
+            BInvokableTypeSymbol invokableTypeSymbol =
+                    createInvokableTypeSymbol(symTag, flags, pkgID, type, owner, pos, origin);
+            if (type instanceof BInvokableType) {
```

Comment:
This if check is unnecessary since we check the type tag above.

Reply:
@pubudu91 without this check, there would be a `SpotBugs` test failure. 

![image](https://user-images.githubusercontent.com/9464997/114139612-e71d4d00-992c-11eb-820c-b515caee7d37.png)

Reply:
Let's add an exception for this warning in the `Symbols` class

Reply:
Added a rule to exclude the `BC_UNCONFIRMED_CAST` in `Symbols` class at SpotBugs excluding XML file.